models/mongo/notes.go

```diff
@@ -0,0 +1,89 @@
+package mongo
+
+import (
+	"accounts-service/models"
+	"errors"
+	"time"
+
+	"github.com/google/uuid"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.uber.org/zap"
+	"golang.org/x/net/context"
+)
+
+type notesRepository struct {
```

Comment:
Same everywhere I guess. `notes` -> `groupNotes`. Let me know if you think this is unnecessary